chrome/html/popup.html

```diff
@@ -1,11 +1,12 @@
-<!doctype html>
-
-<html>  
-    <head>    
-        <title>Test</title>  
-    </head>  
-    
-    <body>    
-        <h1>This is our awesome chrome url shortener</h1>  
-    </body>
+<html>
+<head>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="../css/popup.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Slab" rel="stylesheet">
+</head>
+<body style="height:252;min-width:310px;font-family: 'Roboto Slab', serif;">
```

Comment:
hey nice work overall! I know this is trivial, but wouldn't your html look nicer if you move the style to your css file ?

Reply:
That's true. Its still work in progress. I will be making more additions

Reply:
oh alright.. then